scripts/core/sampler.yml

```diff
@@ -21,15 +47,15 @@ etors:
             The reference count returned should be considered immediately stale. 
             It is unsuitable for general use in applications. This feature is provided for identifying memory leaks.
     - name: CONTEXT
-      desc: "Sampler context info"
+      desc: "[$x_context_handle_t] Sampler context info"
     - name: NORMALIZED_COORDS
-      desc: "Sampler normalized coordindate setting"
+      desc: "[bool] Sampler normalized coordinate setting"
     - name: ADDRESSING_MODE
-      desc: "Sampler addressing mode setting"
+      desc: "[$x_sampler_addressing_mode_t] Sampler addressing mode setting"
     - name: FILTER_MODE
-      desc: "Sampler filter mode setting"
+      desc: "[$x_sampler_filter_mode_t] Sampler filter mode setting"
     - name: MIP_FILTER_MODE
-      desc: "Sampler MIP filter mode setting"
+      desc: "[bool] Sampler MIP filter mode setting"
     - name: LOD_MIN
```

Comment:
I can't find any usage of `LOD_MIN` nor `LOD_MAX` is intel/llvm

Reply:
Interesting, I guess this has been there since creation of this repo?

Maybe we can remove them. Could you check the OpenCL 2.0 spec?

Reply:
It seems they are copied from opencl: CL_SAMPLER_LOD_MIN_KHR and CL_SAMPLER_LOD_MAX_KHR. I cannot find an equivalent for sycl though. So maybe they could be removed?

Reply:
I can't seem to find usage of MIP_FILTER_MODE either. Did you find any? @veselypeta

Reply:
In the event we do actually need these we can add them back in.